scenarios/gaming/README.md

```diff
@@ -0,0 +1,29 @@
+
+### Personalized item recommendation
+
+The addition of micro-transactions inside games have produced a complete shift in the business model of games. Traditionally, most games were based on a one-off purchase, and users got all the content. Nowadays, many games allow players to buy items while they are playing via micro-transactions. Personalized item recommendations can be used to increase the player engagement, tailor the content to the player's interests, and increase the player's retention. In addition, these recommendation provide benefits for studios, which will increase their conversion rate and revenue. These systems also take into account the players past behavior and their interactions with items. The same set of algorithms used in personalized game recommendation can be used for personalized item recommendation, by just adapting the dataset.
+
+### Similar items
+
+A related scenario to personalized item recommendation is similar item recommendations. In this case the recommendations are based on item feature similarity. Some features that can be taken into account are price, item category, item usage, etc. The business usecase for similar items can be increasing the likelihood of buying or avoiding losing a sale by showing an alternative item with a lower price (down-selling). The set of algorithms that can be used in this scenario are those based on content-based filtering, such as [LightGBM](../../examples/00_quick_start/lightgbm_tinycriteo.ipynb), [VW](../../examples/02_model_content_based_filtering/vowpal_wabbit_deep_dive.ipynb) or even [xDeepFM](../../examples/00_quick_start/xdeepfm_criteo.ipynb).
+
+### Next best action prediction
+
+An interesting scenario is next best action prediction. In this scenario, what is recommended is the most beneficial next action for the player. From the technical point of view, this can be implemented using collaborative filtering algorithms, such as [SAR](../../examples/00_quick_start/sar_movielens.ipynb), [BPR](../../examples/02_model_collaborative_filtering/cornac_bpr_deep_dive.ipynb), and [NCF](../../examples/00_quick_start/ncf_movielens.ipynb).
+
+## Data and evaluation
+
+Datasets used in gaming recommendations usually include [user information](../../GLOSSARY.md), [item information](../../GLOSSARY.md) and [interaction data](../../GLOSSARY.md), among others. 

```

Comment:
Is any context based information (e.g. session) available?

Reply:
That's a good point. @angusrtaylor do you know if there is session data in the current engagement you are looking at? I'll ask internally as well

Reply:
There is data on when players logged in, added/removed an item to their inventory, matched with a multiplayer game etc.

Reply:
updated